File: Cuenta_bancaria/Empleado.java
 package Cuenta_bancaria;

import java.util.ArrayList;

public class Empleado{
    
    private String name;
    private ArrayList<BankAccount> cuentas;


    public Empleado(String name) {
       
        this.name = name;
        this.cuentas=new ArrayList<>();
    }
    public String getName() {
        return name;
    }

    public void agregarCuenta(BankAccount account){
        cuentas.add(account);
    }
    
    public ArrayList<BankAccount> getCuentas(){
        return cuentas;
    }

    public BankAccount getAccountPorNumero(long accountNumber) {
        for (BankAccount account : cuentas) {
            if (account.getAccountNumber()==accountNumber) {
                return account;
            }
        }
        return null;
    }

    
    public void verCuentas() {
        System.out.println("Cuentas del empleado " + name + ":");
        for (BankAccount account : cuentas) {
            System.out.println(account);
        }
    }

    
}

File: Isra_Bank/Empleado.py
class Empleado:
    def __init__(self, name):
        self.name = name
        self.cuentas = []

    def get_name(self):
        return self.name

    def agregar_cuenta(self, account):
        self.cuentas.append(account)

    def get_cuentas(self):
        return self.cuentas

    def get_account_por_numero(self, account_number):
        for account in self.cuentas:
            if account.get_account_number() == account_number:
                return account
        return None

    def ver_cuentas(self):
        print("Cuentas del empleado", self.name + ":")
        for account in self.cuentas:
            print(account)